packages/checkout/sdk/src/smartCheckout/routing/routingCalculator.ts

```diff
@@ -188,11 +195,22 @@ export const getOnRampFundingStep = async (
   return onRampFundingStep;
 };
 
+const handleRoutePromise = (
```

Comment:
Only thing Id recommend is maybe tweaking the name to be more contextual to the next/progress callback.. like `handleRouteProgress` or `routeProgress` etc